### PR TITLE
No-SNOW Update fasterxml

### PR DESCRIPTION
**Pull request by sfc-gh-xhuang**
Update to a newer version of fasterxml

### CODE REVIEW

**Review by sfc-gh-asen — APPROVED** [review]
Thank you

**Review by sfc-gh-tzhang — APPROVED** [review]
LGTM, please fix the merge gate before merging

File: pom.xml [hunk 1]
```diff
@@ -723,6 +723,7 @@
                 <banDuplicateClasses>
                   <ignoreClasses>
                     <ignoreClass>META-INF/versions/*/org/bouncycastle/*</ignoreClass>
+                    <ignoreClass>META-INF/versions/*/com/fasterxml/jackson/core/io/doubleparser/*</ignoreClass>
```
Comment (sfc-gh-tzhang) [inline pom.xml:726]: I'm not sure if we want to keep adding to this list to resolve duplicate class issue, I know that we have a long history of issue for boucycastle. I believe what we did before is to exclude the class from one of the package? cc: @sfc-gh-lsembera

**Comment by sfc-gh-xhuang** [comment]
I "fixed" the merge gates by adding an ignoreclass. Is that the right way to do this? 

cc: @sfc-gh-tzhang @sfc-gh-japatel

**Comment by sfc-gh-xhuang** [comment]
Also @sfc-gh-lsembera @sfc-gh-tzhang Do you know where snyk is complaining about commons-configuration2? 
I can't find where we define usage of that

File: pom.xml (hunk 1 above)
Comment (sfc-gh-xhuang) [reply pom.xml:726]: @sfc-gh-lsembera any concerns here?

**Comment by sfc-gh-tzhang** [comment]
> Also @sfc-gh-lsembera @sfc-gh-tzhang Do you know where snyk is complaining about commons-configuration2? I can't find where we define usage of that

Please check with @sfc-gh-lsembera , @sfc-gh-alhuang might also know more on this

**Comment by sfc-gh-lsembera** [comment]
@sfc-gh-xhuang `commons-configuration` 2.8.0 is being pulled in transitively via `hadoop-common`. You can run `mvn dependency:tree` and then check `target/dependency-list.txt`.

**Review by sfc-gh-lsembera — APPROVED** [review]

**Pull request merged**